mitodl/micromasters
# remove max validation from final grade

#### What are the relevant tickets?
fixes #4268 

#### What's this PR do?
Remove max validation on final grade.

#### How should this be manually tested?

- Change [DEFAULT_GRADE](https://github.com/mitodl/micromasters/blob/master/seed_data/management/commands/__init__.py#L11) to ? 1
- Run `alter_data` with `set_to_passed`
- Command should run successfully and grade should be appeared on dashboard.

**P.S** @pdpinch We may want to add argument in `set_to_passed` to pass an optional desired grade instead of default.

#### Any background context you want to provide?
edX allows grades above 100%. Our code does a validation that prevents that. We have to change it.

#### Screenshots (if appropriate)
![screen shot 2019-03-06 at 12 13 36 pm](https://user-images.githubusercontent.com/4245618/53862478-536c4a80-4009-11e9-9776-92baa2e1c682.png)
